Simonyi/Troubleshooting/MTCS/HexRot/MTRot-Motion-Check.rst

```diff
@@ -0,0 +1,97 @@
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: Yijung Kang
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: Kshitija Kelkar
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _MTRot-Motion-Check:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+######################
+MTRotator Motion Check
+######################
+
+
+.. _MTRot-Motion-Check-Overview:
+
+Overview
+========
+
+.. This section should provide a brief, top-level description of the document's purpose and utilization. 
+
+This procedure checks the motion of the MTRotator under nominal movements, issued independently 
+from the MTQue or the MTRotator EUI.
+
+.. _MTRotator-motion-check-script:
+
+Using the `move_rotator` SAL script
+===================================
+
+Use the following configuration to run :file:`maintel/mtrotator/move_rotator.py` using the follwoing configuration 
+
+.. code-block:: SAL
+    :caption: move_rotator.p
+
+    angle = <+ or - angle you want to move in degrees>
+
+If this procedure fails, follow the next one using the MTRotator EUI 
+
+.. _MTRotator-motion-check-eui:
+
+Using the MTRotator EUI
+=======================
+
+MTRotator EUI Access
+--------------------
+
+#.  Enter *hexrot-vm02.cp.lsst.org* with your IPA account credentials.
+    
+#.  Once in the virtual machine, choose your user profile and enter your IPA password.
+
+#.  Open a terminal from the 'Activities' tab on top left and go to the following - 
+
+    .. prompt:: bash
+
+        cd /rubin/
+        cd rotator/
+        cd build
+        ./runRotEui
+
```

Comment:
.. prompt:: bash

        cd /rubin/rotator/build
        ./runRotEui

Reply:
I think first it is needed to check that there are not processes running on the EUI. So I suggest to consider that situation first before cd /rubin/rotator/build and ./runRotEui.
> ps -aux | grep "runRotEui"

If the process is running I suppose we should kill it by ID.

Reply:
Thanks! this is a helpful tip !